src/Microsoft.ApplicationInsights/Extensibility/Implementation/RichPayloadEventSource.TelemetryHandler.cs

```diff
@@ -211,7 +212,8 @@ private Action<ITelemetry> CreateHandlerForRequestTelemetry(EventSource eventSou
                     dummyRequestData.url,
                     dummyRequestData.properties,
                     dummyRequestData.measurements
-                }
+                },
+                PartA_flags = this.dummyPartAFlagsValue,
```

Comment:
https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/Implementation/RichPayloadEventSourceTest.cs  - Can you modify unittests for RichPayloadEventSource to check for flags?